aten/src/ATen/native/mkldnn/Conv.cpp

```diff
@@ -709,6 +725,8 @@ Tensor mkldnn_convolution_transpose_pointwise(
     torch::List<c10::optional<at::Scalar>> scalars,
     c10::optional<c10::string_view> algorithm) {
   c10::impl::ExcludeDispatchKeyGuard edkg(c10::autograd_dispatch_keyset);
+  bool use_channels_last =
+      weight_t.is_mkldnn() || mkldnn_conv_use_channels_last(input_t, weight_t);
```

Comment:
The logic is ok. Not related to this PR, but does it make more sense that `mkldnn_conv_use_channels_last` should prefer channels last if input is a strided tensor and weight is mkldnn? @XiaobingSuper